homework4/Task2.java

```diff
@@ -0,0 +1,29 @@
+package homework4;
+
+//Имеея строка с текстом. Подсчитать количество слов в тексте. Желательно учесть, что слова могут
+//разделяться несколькими пробелами,
+//в начале и конце текста также могут быть пробелы, но могут и отсутствовать.
+
+import java.util.Scanner;
+
+public class Task2 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Введите слова одной строкой через пробел");
+        String input = sc.nextLine();
+        int count = 0;
+        input=input.trim();
+        input=input.replaceAll("\\s{2,}", " ");
```

Comment:
Ok, but it your string will looks like this: "this is the very first,not second,row I wrote in java with fifteen words" ?